frontend/src/_services/authentication.service.js

```diff
@@ -116,7 +116,10 @@ function resetPassword(params) {
 
 function logout() {
   clearUser();
-  history.push(`/login?redirectTo=${window.location.pathname?.startsWith('/sso/') ? '/' : window.location.pathname}`);
+  const loginPath = (window.public_config?.SUB_PATH || '/') + 'login';
+  history.push(
+    loginPath + `?redirectTo=${window.location.pathname?.startsWith('/sso/') ? '/' : window.location.pathname}`
```

Comment:
```suggestion
  history.push(
    loginPath + `?redirectTo=${window.location.pathname}`
```
the start with /sso/ check is no longer required since we have removed log out call from Authorize page